CommonMark/CommonMarkAdditionalFeatures.cs

```diff
@@ -21,6 +21,26 @@ public enum CommonMarkAdditionalFeatures
         StrikethroughTilde = 1,
 
         /// <summary>
+        /// The parser will recognize syntax <c>~foo~</c> that will be rendered as <c>&lt;sub&gt;foo&lt;/sub&gt;</c>.
+        /// </summary>
+        SubscriptTilde = 2,
+
+        /// <summary>
+        /// The parser will recognize both <see cref="StrikethroughTilde"/> and <see cref="SubscriptTilde"/>.
+        /// </summary>
+        Tilde = 3,
+
+        /// <summary>
+        /// The parser will recognize syntax <c>^foo^</c> that will be rendered as <c>&lt;sup&gt;foo&lt;/sup&gt;</c>.
+        /// </summary>
+        SuperscriptCaret = 4,
+
+        /// <summary>
+        /// The parser will treat reference labels as case sensitive.
+        /// </summary>
+        RespectReferenceCase = 0x100,
```

Comment:
Re: `//comment` Should I also add comments to 1, 2 and 4 for consistency? ;)
Re: distant value As insignificant as this option seems on the surface, enabling it introduces a potentially breaking change (i.e. links might stop working due to wrong casing).